easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.0-GCC-5.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.0'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '5.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads',]
+
+dependencies = [('hwloc', '1.11.3')]
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-verbs '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
+configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+
+# needed for --with-verbs
+osdependencies = [('libibverbs-dev', 'libibverbs-devel'), ('libpciaccess-devel'), ('libudev-devel', 'systemd-devel'),]
```

Comment:
I adapted this easyconfig from one produced by someone else at my workplace (from version 1.10.2) so I kept the dependencies the same throughout. Looking at one of the easyconfigs on your repository for Open MPI, though, I can see that it is probably not necessary. I will try a build now with less osdependencies like in that easyconfig and re-push.